src/components/Footer/index.tsx

```diff
@@ -0,0 +1 @@
+export * as Footer from './Footer';
```

Comment:
Тут варто експортувати по інакшому, бо можуть виникнути проблеми при підключенні компонента.
```suggestion
export *  from './Footer';
```